p2panda-js/src/wasm.test.ts

```diff
@@ -100,20 +100,22 @@ describe('WebAssembly interface', () => {
       );
 
       expect(() => fields.add('contact', 'relation', [TEST_HASH])).toThrow(
-        'Invalid object',
+        'Expected a hash value for field of type relation',
```

Comment:
Lolz, was looking at this for a while thinking "But a relation should be just a hash, isn't this wrong?"